nuxeo-core/nuxeo-core-schema/src/main/java/org/nuxeo/ecm/core/schema/DocumentTypeDescriptor.java

```diff
@@ -40,7 +41,10 @@
  * @author <a href="mailto:bs@nuxeo.com">Bogdan Stefanescu</a>
  */
 @XObject("doctype")
-public class DocumentTypeDescriptor {
+public class DocumentTypeDescriptor implements Descriptor {
```

Comment:
Now I do the override correctly. (lost in cherry-picks 11 > 10 > 11)
Had to (re)make a cast in `SchemaManagerImpl` since it relies on a method that returns a `DocumentTypeDescriptor`.